fabtests/component/dmabuf-rdma/util.c

```diff
@@ -126,14 +126,32 @@ void sync_tcp(int sockfd)
 	(void)! read(sockfd, &dummy2, sizeof dummy2);
 }
 
+static int readFully(int fd, void *obuf, int len)
```

Comment:
Please use naming convention consistent with the rest of the file.  For example, 'readFully' --> 'read_fully'. Same for the variable names.